web/api/v1/api.go

```diff
@@ -1954,3 +1955,28 @@ func marshalExemplarJSON(ptr unsafe.Pointer, stream *jsoniter.Stream) {
 func marshalExemplarJSONEmpty(ptr unsafe.Pointer) bool {
 	return false
 }
+
+// marshalLabelsJSON writes.
+//
+//	{
+//	   <labelName1>: <labelValue1>,
+//	   <labelName2>: <labelValue2>,
+//	   ...
+//	   <labelNameN>: <labelValueN>
+//	}
+func marshalLabelsJSON(ptr unsafe.Pointer, stream *jsoniter.Stream) {
+	lbls := *((*labels.Labels)(ptr))
+	stream.WriteObjectStart()
+	for i, lbl := range lbls {
+		stream.WriteObjectField(lbl.Name)
```

Comment:
Interested to know if this version is better than https://github.com/cortexproject/cortex/blob/f19c4016e36c80d5129520903160063bd088c31a/pkg/chunk/json_helpers.go#L42